tools/accuracy_checker/accuracy_checker/adapters/README.md

```diff
@@ -222,3 +222,4 @@ AccuracyChecker supports following set of adapters:
   * `boxes_out` - name of output layer with bounding boxes coordinates.
 * `prnet` - converting output of PRNet model for 3D landmarks regression task to `FacialLandmarks3DPrediction`
     * `landmarks_ids_file` - the file with indeces for landmarks extraction from position heatmap. (Optional, default values defined [here](https://github.com/YadiraF/PRNet/blob/master/Data/uv-data/uv_kpt_ind.txt))
+* `face_detection` - converts output of face detection model to `DetectionPrediction ` representation.
```

Comment:
could you please provide more details about model and its outputs format and/or give more descriptive name for new adapter?
We have several face detection models in OMZ, this common description can confuse users...

Reply:
I have defined FaceDetectionAdapter to be more generic and reflected the changes in README.

Reply:
ok, thanks